src/citrine/resources/material_run.py

```diff
@@ -163,10 +126,20 @@ def get_history(self, scope, id) -> Type[MaterialRun]:
         base_path = os.path.dirname(self._get_path(ignore_dataset=True))
         path = base_path + "/material-history/{}/{}".format(scope, id)
         data = self.session.get_resource(path)
-        # Rehydrate a taurus object based on the data
-        model = loads(json.dumps([data['context'], data['root']], cls=TaurusEncoder))
-        # Convert taurus objects into citrine-python objects
-        return MaterialRun.build(model)
+
+        # Add the root to the context and sort by writable order
+        blob = dict()
```

Comment:
This reconstructs the json serialization format so that json deserialization can be used to build the connected material history